stdlib/public/core/Integers.swift.gyb

```diff
@@ -2361,8 +2395,7 @@ ${operatorComment('&' + x.operator, True)}
 //===----------------------------------------------------------------------===//
 
 /// An integer type that can represent only nonnegative values.
-public protocol UnsignedInteger : BinaryInteger
-  where Magnitude : BinaryInteger { }
+public protocol UnsignedInteger : BinaryInteger { }
```

Comment:
@natecook1000 I didn't notice this change the first time round. Is there a reason that `UnsignedInteger` (and, below, `SignedInteger`) are losing the constraint `where Magnitude : BinaryInteger`?